lib/dropMenu.tsx

```diff
@@ -0,0 +1,91 @@
+              <Image width={6} height={6} className="w-6 h-6 mr-2 rounded-full" src={fav} alt="Jese image" />
+              Charles Leclerc
+            </Link>
+          </li>
+          <li onClick={() => { setSelectCreator("creatorFour");
+                               context.setCreator("FULL SEND MMA"); 
+                               context.setPayment("https://buy.stripe.com/test_eVa2aB7Xj4HPg2Q6ou")
+                               context.setImage("https://yt3.googleusercontent.com/DVtnT7dhsBtGqPYzmzTocgRsGe3Obf86n_hN2aq4zPZNk62tM13Y2CdK0eDe6UKaZwh_teR775c=s900-c-k-c0x00ffffff-no-rj") }}>
+            <Link href="#" className="flex items-center px-4 py-2 hover:bg-sky-500 dark:hover:bg-gray-600 dark:hover:text-white">
+              <Image width={6} height={6} className="w-6 h-6 mr-2 rounded-full" src={fav} alt="Jese image" />
+              FULL SEND MMA
+            </Link>
+          </li>
+          <li onClick={() => { setSelectCreator("creatorFive"); 
+                               context.setCreator("Amy Cuddy");
+                               context.setPayment("https://buy.stripe.com/test_eVaeXndhDdel9EsfZ5")
+                               context.setImage("https://yt3.googleusercontent.com/DVtnT7dhsBtGqPYzmzTocgRsGe3Obf86n_hN2aq4zPZNk62tM13Y2CdK0eDe6UKaZwh_teR775c=s900-c-k-c0x00ffffff-no-rj")  }}>
+            <Link href="#" className="flex items-center px-4 py-2 hover:bg-sky-500 dark:hover:bg-gray-600 dark:hover:text-white">
+              <Image width={6} height={6} className="w-6 h-6 mr-2 rounded-full" src={fav} alt="Jese image" />
+              Amy Cuddy
+            </Link>
+          </li>
+          <li onClick={() => { setSelectCreator("creatorFour"); context.setCreator("Joseph Mcfall"); context.setPayment("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf") }}>
+            <Link href="#" className="flex items-center px-4 py-2 hover:bg-sky-500 dark:hover:bg-gray-600 dark:hover:text-white">
+              <Image width={6} height={6} className="w-6 h-6 mr-2 rounded-full" src={fav} alt="Jese image" />
+              Joseph Mcfall
+            </Link>
+          </li>
+          <li onClick={() => { setSelectCreator("creatorFour"); context.setCreator("Roberta Casa"); context.setPayment("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf") }}>
+            <Link href="#" className="flex items-center px-4 py-2 hover:bg-sky-500 dark:hover:bg-gray-600 dark:hover:text-white">
+              <Image width={6} height={6} className="w-6 h-6 mr-2 rounded-full" src={fav} alt="Jese image" />
+              Roberta Casa
+            </Link>
+          </li>
+          <li onClick={() => { setSelectCreator("creatorFour"); context.setCreator("Neil Sims"); context.setPayment("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf")  }}>
+            <Link href="#" className="flex items-center px-4 py-2 hover:bg-sky-500 dark:hover:bg-gray-600 dark:hover:text-white">
```

Comment:
Why empty Link

Reply:
This was dropMenu from former Creator Page. As with new Creator Page, drop menu is removed ,and probably will not be used unless said otherwise by Franko(I will need to consult with him about that). So for now I just moved it from Creator Page to another file in case if we find better use.  I probably shouldn't even commit that.